tests/cases/setup_patroni/playbook.yml

```diff
@@ -0,0 +1,24 @@
+---
+- hosts: all
+  name: Postgres deployment playbook
+  become: yes
+  gather_facts: yes
+  any_errors_fatal: true
+
+  collections:
+    - edb_devops.edb_postgres
+
+  pre_tasks:
+    - name: Initialize the user defined variables
+      set_fact:
+        disable_logging: false     
```

Comment:
Don't we have to use: `use_patroni: true` ?

Reply:
By default, it is true in `setup_patroni` role. However, I have added it for consistency.